lark/visitors.py

```diff
@@ -458,11 +458,11 @@ class _VArgsWrapper:
     We also have the visit_wrapper attribute to be used by Transformers.
     """
```

Comment:
A simpler solution is to just add this, no?

```python
    base_func: Union[Callable, Callable]      # Union needed because mypy is dumb
```

Reply:
I suspect that the union is a different way to work around the issue I linked to. If you'd rather not change the code, I'd prefer an explicit explicitly ignoring the specific error instead of tricking mypy.

Reply:
Yes, I find the new code less readable. Instance variables should be assigned only once, unless there's a really good reason.

I'm okay with ignoring the specific error.